src/aiida/orm/nodes/data/array/xy.py

```diff
@@ -9,6 +9,13 @@
 """This module defines the classes related to Xy data. That is data that contains
 collections of y-arrays bound to a single x-array, and the methods to operate
 on them.
+
+Example:
+    xy = XyData()
+    xy.set_x(np.array([1, 2, 3]), 'x', 'unit_x')
+    xy.set_y([np.array([1, 2, 3]), np.array([4, 5, 6])], ['y', 'z'], ['unit_y', 'unit_z'])
+    print(xy.get_arraynames())  # ['x_array', 'y_array_0', 'y_array_1']
```

Comment:
Okay, make sense. Could you then also add an example for `get_y_arraynames` here?